account-decoder/src/parse_account_data.rs

```diff
@@ -31,7 +30,9 @@ lazy_static! {
         );
         m.insert(*CONFIG_PROGRAM_ID, ParsableAccount::Config);
         m.insert(*SYSTEM_PROGRAM_ID, ParsableAccount::Nonce);
-        m.insert(*TOKEN_PROGRAM_ID, ParsableAccount::SplToken);
+        for spl_token_id in spl_token_ids() {
+            m.insert(spl_token_id, ParsableAccount::SplToken);
```

Comment:
Here I'm assuming that all newer SPL Token instances will be supersets of older instances, so it's ok to reuse the same parsers for all